sdk/cognitiveservices/cognitiveservices-luis-authoring/package.json

```diff
@@ -19,20 +19,13 @@
   "module": "./esm/lUISAuthoringClient.js",
   "types": "./esm/lUISAuthoringClient.d.ts",
   "devDependencies": {
-    "@types/chai": "^4.2.0",
-    "@types/mocha": "^5.2.7",
-    "mocha": "^6.1.4",
-    "nock": "^10.0.0",
-    "rollup": "^0.66.2",
-    "@azure/ms-rest-azure-js": "^2.0.1",
-    "rollup-plugin-node-resolve": "^3.4.0",
+    "typescript": "^3.5.3",
+    "rollup": "^1.18.0",
+    "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-sourcemaps": "^0.4.2",
-    "ts-node": "^8.3.0",
-    "typescript": "^3.1.1",
-    "uglify-js": "^3.4.9",
-    "ts-mocha": "^6.0.0"
+    "uglify-js": "^3.6.0"
   },
-  "homepage": "https://github.com/azure/azure-sdk-for-js/tree/main/sdk/cognitiveservices/cognitiveservices-luis-authoring",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js",
```

Comment:
I believe we want to keep the older homepage here.

@sarangan12, Can you please share if this is doable using the code generator or should the user make this change manually?